content/cpp/concepts/files/files.md

```diff
@@ -89,3 +89,56 @@ The output would be:
 Today is the greatest
 Day I've ever known
 ```
+
+## Appending to a File
+In some cases, you may want to append text to an existing file instead of overwriting its content. To do this, you can open the file in append mode using the `std::ios::app` flag with `ofstream` or `fstream`. Here's an example:
+
+```cpp
+#include <iostream>
+#include <fstream>
+int main() {
+  // Open the file in append mode
+  std::ofstream MyFile("journal.txt", std::ios::app);
+  // Append text to the file
+  MyFile << "\nSmashing Pumpkins lyrics\n";
+  // Close the file
+  MyFile.close();
+}
+```
+
+The output would be:
```

Comment:
In this case, there won’t be any console output. The program appends text to the file journal.txt, but it doesn’t produce output to the console.

So, we should rather write something like this - 

```suggestion
The file `journal.txt` will now contain:
```